assets/core/leader-ingress-to-route-controller-role.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: system:openshift:openshift-controller-manager:leader-locking-ingress-to-route-controller
```

Comment:
this role + binding should not be needed anymore and will be removed from OCM-o as well